dist/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/5e67ba493b.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Todo List</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <form class="todo-form">
+      <div class="todo-header">
+        <h2>Today's To Do</h2>
```

Comment:
- [ ]  I kindly suggest that you use `<h1>` tag, since it is recommended to to do so in other adhere `HTML` semantic rules since is the first heading

Reply:
Thank you for this feedback. Really helpfull👌